lib/resty/cassandra/cluster.lua

```diff
@@ -920,7 +920,10 @@ local function handle_error(self, err, cql_code, coordinator, request)
     end
   elseif err == 'timeout' then
     if self.retry_on_timeout then
-      return self:send_retry(request, 'timeout')
+      local should_retry = self.retry_policy:on_connect_timeout(request)
```

Comment:
shouldn't this be 'on_socket_timeout'?

Reply:
config that is used for socket timeout is called `timeout_connect` that is why I renamed function